atom/browser/browser.cc

```diff
@@ -156,6 +156,8 @@ void Browser::WillFinishLaunching() {
 
 void Browser::DidFinishLaunching(const base::DictionaryValue& launch_info) {
   is_ready_ = true;
+  sb_service = safe_browsing::SafeBrowsingService::CreateSafeBrowsingService();
```

Comment:
I think this should go in chromium_src/chrome/browser/browser_process_impl.cc to match the implementation in src/chrome/browser/browser_process_impl.cc

Reply:
I am working on the changes, I realized this yesterday. This is still in progress. Should've added the WIP tag to the header.